app/server.go

```diff
@@ -477,7 +477,7 @@ func NewServer(options ...Option) (*Server, error) {
 		return nil, errors.Wrap(err, "Unable to create opengraphdata cache")
 	}
 
-	s.createPushNotificationsHub()
+	s.createPushNotificationsHub(request.EmptyContext())
```

Comment:
We are just passing empty context, which means there is no request_id attached. Wondering if in all these methods, we can just use `s.Log` or `app.Srv().Log`? I was thinking that we should just use contexts for request-based methods, and not for long-running code like the web hub.

Reply:
I need to check for this particular instance, but I think it calls methods that require a context now.

Reply:
Ok, do take a look. But overall, I am thinking we don't need to pass the logger for methods that aren't part of the request flow.

Reply:
So this indirectly calls `IsCRTEnabledForUser` which calls the preference store. There are no log statements in this call but I thought it would be nice to pass the context into it anyway, since there may be more data added to the context in time. That said, I'm 1/5 whether to keep it or remove it again.

Reply:
Yep sounds good. Let's initialize EmptyContext with the existing logger as commented below.